base/stats.go

```diff
@@ -16,13 +52,39 @@ const (
 //to find valid property names
 //
 func init() {
-	StatsExpvars.Add("changesFeeds_total", 0)
-	StatsExpvars.Add("changesFeeds_active", 0)
-	StatsExpvars.Add("requests_total", 0)
-	StatsExpvars.Add("requests_active", 0)
-	StatsExpvars.Add("revisionCache_hits", 0)
-	StatsExpvars.Add("revisionCache_misses", 0)
+	for _, stat := range globalStatNames {
+		StatsExpvars.Add(stat, 0)
+	}
 	TimingExpvars = NewSequenceTimingExpvar(KTimingExpvarFrequency, KTimingExpvarVbNo, "st")
 	StatsExpvars.Set("sequenceTiming", TimingExpvars)
+	InitDbStats("_server")
+
+	ticker := time.NewTicker(statLogInterval)
+	go func () {
+		for {
+			<-ticker.C
+			Statsf(KeyAll, StatsExpvars.String())
```

Comment:
expvar.Map.String() is relatively expensive - it's building JSON by iterating over the map, obtaining a read lock for each entry and serializing it.  We get this performance hit per stat, whether or not that stat has changed.